vic/drivers/image/src/vic_image_start.c

```diff
@@ -45,6 +45,17 @@ vic_image_start(void)
         get_global_param(filep.globalparam);
     }
 
+    if (mpi_rank == VIC_MPI_ROOT) {
+        // Open domain and parameter netCDF files
+        filenames.domain.nc_id = open_nc(filenames.domain.nc_file);
+        filenames.params.nc_id = open_nc(filenames.params.nc_file);
+    }
+
     // initialize image mode structures and settings
     vic_start();
+
+    if (mpi_rank == VIC_MPI_ROOT) {
+        // Close domain file
+        close_nc(filenames.domain);
+    }
```

Comment:
Yes we can do this opening and closing domain file thing entirely inside `vic_start`. Do you think we should do the same thing for opening and closing parameter file as well (although we do need to open the parameter file in `vic_start`, and close it in `vic_init`, if that's what we want to do, since the parameter file variables are used in both functions)?